New version: EmbeddingsTools v0.2.2

JuliaRegistrator opened the pull request:
- Registering package: EmbeddingsTools
- Repository: https://github.com/Marwolaeth/EmbeddingsTools.jl
- Created by: @Marwolaeth
- Version: v0.2.2
- Commit: f3bda1a37578e7d90d03c97fe4874e3d717feae2
- Reviewed by: @Marwolaeth
- Reference: https://github.com/Marwolaeth/EmbeddingsTools.jl/issues/2#issuecomment-1935483213
- Description: Extra tools for working with word embeddings, such as those in Embeddings.jl. However, the compatibility is currently limited.
<!-- bf0c69308befbd3ccf2cc956ac8a4671113b158fd87a6d4e741b0173dab051a48f7dc43e593ffff4357bc647a3a83598525549fc6248f3ede3c1a10752e325e12b796f45bd8657fd3480fad41c7a86e00b78163af47b17b1e27a1fe18c860c6412f9f945e871427df0e34931355c30e08e1251e2753586377c00a0d909bafc0712e6117f545604b45e14b3170c710ddd11f38c690ecc05633d01aeb793875bf84f035a26d72e53be88766667269deef2 -->

DilumAluthge commented:
[noblock]

Looks like there is a merge conflict.

@Marwolaeth Can you re-trigger Registrator? This will fix the merge conflict.

JuliaRegistrator committed:
New version: EmbeddingsTools v0.2.2

UUID: 81c6ba3f-3028-455b-8a86-eff211c369f5
Repo: https://github.com/Marwolaeth/EmbeddingsTools.jl.git
Tree: 9b46805ef514c456d49718124707b37295a08204

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
